frontend/src/features/SideWindow/MissionForm/MainForm/FormikMultiControlUnitPicker/ControlUnitSelect.tsx

```diff
@@ -194,6 +204,7 @@ export function ControlUnitSelect({
         />
         <TextInput
           disabled={!controlUnits || !controlledValueRef.current.name}
+          isUndefinedWhenDisabled
```

Comment:
On peut pas mettre ça par défaut ?

Reply:
Alors c'était par défaut avant mais ça a posé des problèmes à Thomas de ne pas avoir une option pour le disable car il utilise des input disabled pour afficher des informations apparemment. Du coup comme je préfère les options "positive" plutôt que "négative" et qu'on a besoin de ce genre de chose que sur certains champs, on s'est mis d'accord pour le passer en option en mode "opt-in".